instrumentation/google_apis_core/lib/opentelemetry/instrumentation/google_apis_core/patches/http_command.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module GoogleApisCore
+      module Patches
+        # Module to prepend to ::Google::Apis::Core::HttpCommand for instrumentation
+        module HttpCommand
```

Comment:
Module is monkey patching https://github.com/googleapis/google-api-ruby-client/blob/google-apis-core/v0.2.1/google-apis-core/lib/google/apis/core/http_command.rb#L358-L406

Reply:
I would strongly prefer just to edit google-apis-core itself to provide whatever explicit hooks are needed. Or perhaps even to build the instrumentation directly into google-apis-core, the way that it currently has opencensus built in. So we can avoid monkey patches. (I effectively own google-apis-core myself, so I can fast-track any such work and release.)